Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -65,60 +65,152 @@ $LogFile = "ExchAvExclusions.log"
 # Open log file if switched
 if ($OpenLog) { Write-SimpleLogFile -OpenLog -String " " -Name $LogFile }
 
+$serverExchangeInstallDirectory = Get-ItemProperty HKLM:\SOFTWARE\Microsoft\ExchangeServer\v15\Setup -ErrorAction SilentlyContinue
+
+# Check Exchange regsitry key
+if (-not  $serverExchangeInstallDirectory ) {
+    Write-Warning "Failed to find the Exchage instalation Path registry key"
+    exit
+}
+
+# Check the installation path
+if (-not ( Test-Path $($serverExchangeInstallDirectory.MsiInstallPath) -PathType Container) ) {
+    Write-Warning "Failed to find the Exchage instalation Path"
+    exit
+}
+
+# Check Exchange is 2016 or 2019
+if ( -not ( $($serverExchangeInstallDirectory.MsiProductMajor) -eq 15 -and `
+        ( $($serverExchangeInstallDirectory.MsiProductMinor) -eq 1 -or $($serverExchangeInstallDirectory.MsiProductMinor) -eq 2 ) ) ) {
```

Comment:
Why can't this script be run against 2013? We still need to support that till April 2023.